README.md

```diff
@@ -73,3 +73,11 @@ Participation in the Argo CD project is governed by the [CNCF Code of Conduct](h
 1. [Getting Started with ArgoCD for GitOps Deployments](https://youtu.be/AvLuplh1skA)
 1. [Using Argo CD & Datree for Stable Kubernetes CI/CD Deployments](https://youtu.be/17894DTru2Y)
 
+
+## Vendor-sponsored community resources
```

Comment:
```suggestion
## Vendor-hosted community resources
```
hosted might make it clearer that the vendor isnt directly sponsoring the project, but rather just hosting the content in question.

Reply:
Committed since @balajisiva allowed maintainer edits. @balajisiva lmk if the change is objectionable!